styles/components/_messages.scss

```diff
@@ -43,12 +43,6 @@
         max-width: rem(600);
         margin-top: rem(100);
     }
-
-    .loading-message {
-        font-size: rem(36);
-        line-height: rem(45);
-        color: $color-gray-light;
-    }
     .usda-loading-animation {
```

Comment:
Shouldn't we be putting this on `.loading-message`?

Reply:
We do, in line 96. The current css has two of them and the reason this is deleted.